pkg/utils/http.go

```diff
@@ -0,0 +1,22 @@
+package utils
+
+import (
+	"net/http"
+
+	"github.com/pkg/errors"
+)
+
+// IsHTTPResponseCodeEquals checks whether the returned response from the url matches the status code.
+func IsHTTPResponseCodeEquals(url string, statusCode int) error {
```

Comment:
I feel like this method should return a boolean. When "Is" is used in the method name I would expect to get a true or false return value.